### ISSUE TITLE
All Screens called "Unknown screen"?

### ISSUE THREAD

**Issue by chrisspiegl** [wulkano/Aperture#78]
I am using Aperture through Aperture-Node to create simple screen recordings.

When I run the code to list all the available screens, all of them show up as `Unknown screen`.

```js
[ { id: 1, name: 'Unknown screen' } ]
```

Is there a way to improve this so that it shows something more valuable?

I am on macOS 12.3 with a M1 Max machine.

**Comment by chrisspiegl** [comment]
I found two related node.js packages. Maybe these help bring the correct names over?

- https://github.com/codebytere/node-mac-displays
- https://github.com/kristian/displays

I chose to go with the `node-mac-displays` in my own project for now. The `id` this library provides is equal to the one provided by `aperture-node` so it's compatible and it spits out human friendly names for the displays, like:

- Built-in Retina Display
- DELL P2411H

Possibly this lib could be used to achieve this in this lib as well? Or possibly copy the approach taken by the lib?
